### ISSUE TITLE
Field of view handles and layer interaction

### ISSUE THREAD

**Issue by MrPus** [Rolisteam/rolisteam#55]
[![Build Status](http://www.rolisteam.org/sites/default/files/pixture_reloaded_logo.png)](http://www.rolisteam.org)

Rolisteam version: 1.8.0
Operating System and version: 
-Windows 7

Rolisteam Setting: 
-GM and Server

#What did you do ? (Explain each step from starting rolisteam to the issue)
-Add a character on a vectorial map on any layer
-Change the layer the character is on
-Do not change the active layer and keep the character selected
-Use the "handles" controling the size of the field of view

#What did you expect to see ?
-The handles controling the length and width of the field of view should keep controling it or it should be impossible to move them until I select the layer the character is on.

#What did you get instead ? (Describe in details the issue) 
-We still can move the handles freely but the field of view does not change until the correct layer is selected.

**Comment by MrPus** [comment]
It actually also applies to players who want to move their characters (or change the field of view) while the GM is using another layer.

**Comment by obiwankennedy** [comment]
In other terms, I have to hide the handlers when the edition layer is not the one of the character?

**Comment by MrPus** [comment]
Yes, it might be solved if the handler is hidden or if the characters get deselected when the active layer is changed but this solution might prevent players from selecting their characters.

The GM choosing the active layer also prevent the players from using their pawn. Could it be changed only locally? This might solve the problem for the players.
